README.md

```diff
@@ -43,6 +43,10 @@ Note: GKE or Anthos natively support injecting workload identity for pods.  This
         #   Note: This value can be overwritten if specified in the pod 
         #         annotation as shown in the next step.
         cloud.google.com/token-expiration: "86400"
+
+        # optional: Defaults to 0 if not set. This value defines the container security context with runAsUser
+        #           with the defined user. This could avoid problems related with root requirement from gcloud image
+        cloud.google.com/gcloud-run-as-user: "1000"
```

Comment:
Could you align this explanation to the latest implementations?? 🙇

Reply:
Sure! I have removed the first part as `Default is not set if not set` sounds weird to me. Optional is enough I guess